docs/post-mortem.md

```diff
@@ -0,0 +1,168 @@
+não só no nível pessoal mas também no coletivo. As metodologias ágeis ajudam o 
+desenvolvimento de um bom produto de software, assim como um bom trabalho em equipe
+- Não adianta a ideia parecer interessante para a equipe de desenvolvimento
+se o usuário não vê valor nela. Nosso trabalho é atender as necessidades dos
+clientes e usuários
+- No fase de planejamento deve ser considerado o ritmo do cliente. Não adianta 
+planejar muita funcionalidade se ele só consegue validade metade delas ou menos
+- O fluxo consistente de entrega de funcionalidades e validação constrói um bom 
+relacionamento entre equipe de desenvolvedores e cliente e estimula confiança
+entre todos os envolvidos
+- É importante manter o projeto organizado, planejado e bem estruturado. Isso 
+oferece apoio em diferentes fatores dentro do ciclo de desenvolvimento de software, 
+além de ser algo estabelecido e um combinado firmado com o cliente, 
+o que não permite uma oscilação de escopo, principalmente para projetos com o 
+tempo bem estabelecido e que não seja flexível, também ajuda no entendimento do 
+valor e do que está sendo entregue para o seu cliente
+
+## Para os próximos desenvolvedores
+
+### Recomendações Futuras
+
+- Garantam que o SonnarCloud esteja coletando todas as 12 métricas pedidas pelo professor
+- Investam muito esforço já no começo para configurar o Sonar com todas as métricas
+- Acompanhem a disciplina com um material online, provavelmente cursos gratuitos
+- Não criem os repositórios de vocês forkando os nossos repositórios. Se vocês fizerem
+isso fica _muito fácil_ abrir uma PR pro repositório errado
+no youtube para que não passem tanta dificuldade na hora de desenvolver
+- Analisar melhores maneiras de validação com os clientes, para tentar obter
+respostas mais rápidas e reduzir a possibilidade de atrasos por parte deles
+- Ter uma reunião separada da reunião principal somente para a validação de 
+requisitos e protótipos
+- Sempre estabelecer um prazo para os clientes validarem as funcionalidades entregues
+- Para os desenvolvedores, tenham sempre um tempo extra de segurança para a 
+validação por parte dos POs para não depender da pontualidade deles
+- Busquem consistemente validar as histórias de usuário para que não sobrecarregue o time
+- Para os de MDS, procurem buscar mais autonomia o quanto antes
+- A depender de quantas novas funcionalidades forem levantadas, foquem bastante 
+em refatorações, seja de código ou na interface do sistema, para melhorar a 
+manutenibilidade do projeto, a experiência do usuário com o produto e tornar o 
+código de fácil entendimento para a futura equipe de software que será 
+responsável no DNIT
+- Para os gerentes, fiquem sempre atentos às demandas dos desenvolvedores, porque
+eles são alunos e são comprometidos também por diversas matérias
+- 👽 Busquem conhecimento
+- Tentar envolver os dois POs também na resposta aos formulários de aceitação.
+- Otimizar o LI para não perder tempo hábil
```

Comment:
```suggestion
- Otimizar o Lean Inception para não perder tempo hábil
```
Só se usa siglas caso já tenha sido detalhada anteriormente no texto - Lean Inception (LI) - ou se for uma sigla já muito bem estabelecida como UnB